pages/api/submit.tsx

```diff
@@ -117,28 +117,29 @@ const processFormData = async (
       return await setTimeout(() => res.status(200).json({ received: true }), 1000);
     }
 
-    for (const [key, value] of Object.entries(files)) {
+    for (const [_key, value] of Object.entries(files)) {
       const fileOrArray = value;
       if (!Array.isArray(fileOrArray)) {
         if (fileOrArray.name) {
-          const { isValid, result } = await pushFileToS3(fileOrArray, fileOrArray.name);
-          if (isValid) reqFields[key] = result;
-          else throw new Error(result as string);
+          logMessage.info(`uploading: ${_key} - filename ${fileOrArray.name} `);
+          const { isValid, key } = await pushFileToS3(fileOrArray);
+          if (isValid) {
+            uploadedFilesKeyUrlMapping.set(fileOrArray.name, key);
+            reqFields[_key] = key;
```

Comment:
Added return value (file path) to the reqForms form field property otherwise the value would never be passed to the submission lambda function.